clwb/src/com/google/idea/blaze/clwb/run/BlazeDebuggerKind.java

```diff
@@ -53,7 +53,8 @@ public static BlazeDebuggerKind byDefaultToolchain() {
 
     return switch (kind) {
       case BUNDLED_GDB, CUSTOM_GDB -> gdbBundledOrServer();
-      case BUNDLED_LLDB -> BUNDLED_LLDB;
+      // case BUNDLED_LLDB, CUSTOM_LLDB, use default for SDK compat reasons
+      default -> BUNDLED_LLDB;
```

Comment:
I was going to mention here that your `ROSETTA_GDB` will fall into the wrong branch logically (though it can't happen in runtime here)

Reply:
thus in my implementation it was like 
https://github.com/ujohnny/intellij/blob/a89338e9e868b64e53178aaa7d116bcbcaf53ce4/clwb/src/com/google/idea/blaze/clwb/run/BlazeDebuggerKind.java#L54